packages/x-data-grid-premium/src/components/GridAggregationHeader.tsx

```diff
@@ -41,12 +41,9 @@ const GridAggregationFunctionLabel = styled('div', {
 })<{ ownerState: OwnerState }>(({ theme }) => {
   return {
     fontSize: theme.typography.caption.fontSize,
-    lineHeight: theme.typography.caption.fontSize,
-    position: 'absolute',
-    bottom: 4,
-    fontWeight: theme.typography.fontWeightMedium,
-    color: (theme.vars || theme).palette.primary.dark,
-    textTransform: 'uppercase',
+    lineHeight: 'normal',
```

Comment:
I'm purposefully deviating from the `theme.typography.caption.lineHeight` value here because it is too large. `normal` line-hieght should keep the spacing looking ok between the column header title and aggregation label I think.

If users want to override it, they have the class I guess?

Reply:
Agreed, but I'm only worried if some existing users' styles would break with this.
It's obviously not very likely but theoretically, some users may be using `theme.typography.caption.lineHeight` to override this style.

It's a nitpick though. Feel free to merge it in current shape.